osh/hub/osh_xmlrpc/scan.py

```diff
@@ -262,9 +267,15 @@ def find_tasks(request, query):
         tasks = Task.objects.filter(label__regex=regex)
     if states:
         tasks = tasks.filter(state__in=states)
-    if tasks is not None:
-        result = list(tasks.order_by("-dt_finished").values_list(
-            "id", flat=True))
+    if tasks.exists():
```

Comment:
This, unfortunately, adds another needless query evaluation and will also crash if the `tasks` variable is not reassigned and still points to `None`.

Reply:
Related docs for context: https://docs.djangoproject.com/en/5.0/ref/models/querysets/#django.db.models.query.QuerySet.exists

Reply:
I'm confused how would `tasks` points to `None`, I think it's only possible if none of [`nvr`, `package_name`, `regex`] was passed in - which means it's an invalid function call according to `cmd_find_tasks.Find_Tasks.run`.

Anyway I've updated the code snippet to reflect this potential failure. Commit message also amended, thanks for the review.